package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "final-project",
+  "version": "0.1.0",
+  "description": "Final Project for SEI-39. A website for Developers to show off code and projects they have worked on via github repos and images.",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "bulma": "^0.9.1",
```

Comment:
Adding the Bulma framework is outside the scope of the issue.
We need to reach a group decision before adding a new library so, at a minimum, people know why, when, how to use it, and also so we know that everyone *wants* to use it.

Reply:
Change made